[#31]: Add brackets to import, to prevent "ambiguous indirect export:…

schipperjeroen opened the pull request:
Solves issue #31

patzick commented:
hey @schipperjeroen 
thank you for your PR, unfortunately this solution causes another error, we already tried this some time ago. `html-to-vue` is a cjs export and that's why even `@ts-ignore` directive is there.
I'll take a closer look this week and will get back to you!

schipperjeroen commented:
Hi @patzick ,

Updated the import with how the readme of `html-to-vue` does is. You're correct about the `@ts-ignore`. Added that also. What is the other error? Couldn't see any errors in console or the watcher?

patzick commented:
hey @schipperjeroen!
Thank you very much for contributing! @mdanilowicz moved the code temporarily to our repo to avoid this error, but we'll be looking for a better solution in future!
Closing this one, but we're grateful for your input! 🙌
